lib/stagehand/staging/commit.rb

```diff
@@ -45,13 +48,12 @@ def start_operation.subject=(record)
 
       # Sets the commit_id on all the entries between the start and end op.
       # Returns the commit object for those entries
-      def self.end_commit(start_operation)
+      def self.end_commit(start_operation, options)
         end_operation = CommitEntry.end_operations.create(:session => start_operation.session)
 
-        CommitEntry
-          .where(:id => start_operation.id..end_operation.id)
-          .where(:session => start_operation.session)
-          .update_all(:commit_id => start_operation.id)
+        scope = CommitEntry.where(:id => start_operation.id..end_operation.id, :session => start_operation.session)
+        scope = scope.where('table_name NOT IN (?) OR table_name IS NULL', options[:except]) if options[:except].present?
```

Comment:
I know right? I struggled to make this less ugly but couldn't come up with anything.